yt_dlp/extractor/chzzk.py

```diff
@@ -40,7 +40,7 @@ def _real_extract(self, url):
             note='Downloading channel info', errnote='Unable to download channel info')['content']
 
         if live_detail.get('status') == 'CLOSE':
-            raise ExtractorError('The channel is not currently live', expected=True)
+            raise UserNotLive()
```

Comment:
```suggestion
            raise UserNotLive(video_id=channel_id)
```

Reply:
Thank you for your review.
Chzzk does not allow two channels with the same name to exist. If it can get the channel name, can it display the channel name instead of the channel id?
The 32-character hex string may be difficult to read. This is already the case in #9208 , so I wonder if I need to change it back to the channel ID.

Reply:
I think this is OK. The hex string is in the URL